portfolio/src/main/webapp/index.html

```diff
@@ -59,80 +60,242 @@
+                                suggest the IP of the <strong>Boot Master</strong> that is atacking the network. 
+                            </p>
+                            <div class="rowBan">
+                                <div class="column3">
+                                    <img src="images/proy4/datastru.PNG" class="band3" style="width:100%">
+                                </div>
+                                <div class="column3">
+                                    <img src="images/proy4/ip.PNG" class="band3" style="width:100%">
+                                </div>
+                                <div class="column3">
+                                    <img src="images/proy4/ipatcl.png" class="band3" style="width:100%">
+                                </div> 
+                            </div>
+                        </div>
+                    </div>
+                    </p>
+                </section>
+                <section id="servlet" style="padding-top: 20px; margin: 10px 0 0 0;">
+                    <div class="first-in-column" style="text-align: center;">
+                        <h3 style="color: #eee;">Get your birth-year!</h3>
+                        <p style="color: #eee">
+                            Enter your age! 
+                            <input type="number" min="0" step="1" oninput="validity.valid||(value='');" id="myage"> <br>
+                            Click the "Try it" button to get your birth year!
+                            <button id="agebt" onclick="myAge(), showServerTime();">Try it</button><br><br>
+                            If you already had your birthday, it means you were born in:<p id="demo"></p>
+                            <p style="color: #eee;">As of today:</p> <p id="date-container"></p>
+                        </p>
+                    </div>
+                    <div class="second-in-column" style="text-align: center;">
+                        <h3 style="color: #eee;">Get one of my favorites Quotes!</h3>
+                        <p style="color: #eee;">
+                            Click the "QUOTE" button!
+                            <button id="quo" onclick="addRandomQuote()">QUOTE</button><br><br><br>
+                            <p id="quote-container"></p>
+                        </p>
+                    </div>
+                </section>
+            </div>
+        </div>
+        <div class="popup">
+            <div id="alert">
+                <span onclick="this.parentElement.style.display='none';">
+                    <button class="closebtn" onclick="opacity()">&times;</button>
+                    <strong>IMPORTANT MESSAGE!</strong> This website has been designed to be visited from
+                    <strong></strong>computers</strong>. Design may vary on other devices... (Close this alert!)
```

Comment:
After refreshing the page a couple of times this alert gets a little annoying...  I can think of a couple of things to make it a better experience:

1. Don't show the alert if someone is viewing the page on a desktop computer (can look at something like https://developer.mozilla.org/en-US/docs/Web/CSS/Media_Queries/Using_media_queries)
2. For other devices, still show the alert at the top of the page but also show the rest of the page so a visitor does not need to click on the X button

Reply:
Okay!!! Thank on the feedback